partner_risk_insurance_security/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_full_access_user = fields.Boolean(
+        string="Is Full Access User",
+        compute="_compute_is_full_access_user",
+        store=False,
+    )
+
+    @api.depends("user_id")
+    def _compute_is_full_access_user(self):
+        for partner in self:
+            partner.is_full_access_user = (
+                self.env.user
+                in self.env.ref(
+                    "partner_risk_insurance_security.group_full_access_credit_policy_state"
+                ).users
+            )
```

Comment:
**suggestion**: I think `self.env.user.has_group("partner_risk_insurance_security.group_full_access_credit_policy_state")` is simpler to write and read, and probably also slightly better in terms of performance.